main.rb

```diff
@@ -19,14 +19,20 @@ def main
 
 def display
   puts 'Library Management System'
-  puts '1. List all books'
-  puts '2. List all people'
-  puts '3. Create a person'
-  puts '4. Create a book'
-  puts '5. Create a rental'
-  puts '6. List rentals for a person'
-  puts '7. Quit'
-  print 'Select an option: '
+  menu = {
+    1 => 'List all books',
+    2 => 'List all people',
+    3 => 'Create a person',
+    4 => 'Create a book',
+    5 => 'Create a rental',
+    6 => 'List rentals for a person',
+    7 => 'Quit'
+  }
+  menu.each do |index, value|
+    puts "#{index}. #{value}"
+    puts 'Select an option: '
+  end
```

Comment:
- [ ] Kindly `Select an option:` is being printed on each option, please fix this to replicate the given video 👍 

### Your UI
![image](https://github.com/Santosh-Konappanavar/school_library/assets/102757126/0fdc289d-5af1-4d62-9765-8713a38c2eb7)

### The given video
[Example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ)